src/components/PageTitleTag.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {theme } from 'antd';
+
+const { useToken } = theme;
+
+interface Props {
+  mainText: string;
+  subText?: string;
+}
+const PageTitleCards: React.FC<Props> = ({ mainText, subText }) => {
```

Comment:
should we use PageTitleCard instead of PageTitleCards?

Reply:
I change the name to "PageTittleTag" to be as the same as the name of the component

Reply:
Make sense